src/utilities/getIntrospectionQuery.ts

```diff
@@ -223,6 +230,7 @@ export interface IntrospectionObjectType {
   readonly interfaces: ReadonlyArray<
     IntrospectionNamedTypeRef<IntrospectionInterfaceType>
   >;
+  readonly isOneOf: boolean;
```

Comment:
looks like this was added to the `IntrospectionObjectType` here in addition to the `IntrospectionInputObjectType` below—is that intentional? The other references in this diff seem to only care about input objects.

Reply:
You're so right. Removed in [633331a](https://github.com/graphql/graphql-js/pull/4078/commits/633331aded549b3db00296646ad4833b5f8b5c7f)